hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -603,6 +606,55 @@ public int getRange() {
     }
 
   }
+  
+  /**
+   * Schema Registry codes are compatible with Confluent Schema Registry v5.3.1
+   * https://docs.confluent.io/5.3.1/schema-registry/develop/api.html
+   */
+  public enum SchemaRegistryErrorCode implements RESTErrorCode {
```

Comment:
@kai-chi Have you tried building hopsworks without -DskipTests so we can see if unit tests work on for this Enum?

Reply:
it will skip these tests for instances of this class